<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTable</class>
 <widget class="QWidget" name="DataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DataTable</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="minimumSize">
      <size>
       <width>850</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message Content</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sent Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Check</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pagination" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="infoButton">
        <property name="text">
         <string notr="true">Info</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="chartButton">
        <property name="text">
         <string notr="true">Chart</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="searchButton">
        <property name="text">
         <string notr="true">Search</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDate</set>
        </property>
        <property name="placeholderText">
         <string>Search...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="prevButton">
        <property name="text">
         <string notr="true">Prev</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pagesLabel">
        <property name="text">
         <string notr="true">Pages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextButton">
        <property name="text">
         <string notr="true">Next</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
